test/game-test.js

```diff
@@ -1,20 +1,22 @@
-const chai = require('chai');
-const expect = chai.expect;
-const Game = require('../game.js');
+// const chai = require('chai');
```

Comment:
If you are removing code, I would remove the code altogether and avoid submitting commented out code. Especially in this case where you have the bones of the testing suite - just seems unnecessary.